grails5 update - geb issue

niravassar opened the pull request.

Commit:
grails5 update - geb issue

sdelamo commented on complete/build.gradle:
```diff
@@ -100,7 +100,10 @@ bootRun {
 //tag::webdriverBinaries[]
 webdriverBinaries {
-    chromedriver "${chromeDriverVersion}"
+    chromedriver {
```
this should be same version as the one set in the grails_i18n guide

sdelamo commented on complete/build.gradle:
```diff
@@ -100,7 +100,10 @@ bootRun {
+        version = "${chromeDriverVersion}"
+        architecture = 'X86'
+    }
     geckodriver "${geckodriverVersion}"
```
this should be same version as the one set in the grails_i18n guide

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,11 +1,15 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
```
grails version should be 5.0.1

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,11 +1,15 @@
-grailsVersion=4.0.1
-gormVersion=7.0.2.RELEASE
+grailsVersion=5.0.0
+gormVersion=7.1.0.M5
```
this should be GORM 7.1.0

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,11 +1,15 @@
 webdriverBinariesVersion=2.1
-hibernateCoreVersion=5.4.0.Final
+hibernateCoreVersion=5.5.7.Final
 chromeDriverVersion=77.0.3865.40
```
this should be same version as the one set in the grails_i18n guide

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,11 +1,15 @@
-hibernateCoreVersion=5.4.0.Final
+hibernateCoreVersion=5.5.7.Final
 chromeDriverVersion=77.0.3865.40
 geckodriverVersion=0.24.0
```
this should be same version as the one set in the grails_i18n guide

sdelamo commented on complete/gradle.properties:
```diff
@@ -1,11 +1,15 @@
+org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
 gebVersion=3.2
 seleniumVersion=3.141.59
 webdriverBinariesVersion=2.1
```
this should be 2.4

sdelamo commented on complete/build.gradle:
```diff
@@ -8,7 +8,7 @@ buildscript {
 //tag::buildscriptDependenciesWebdriverBinaries[]
```
Grails Gradle plugin version should be decoupled from grails version

jamesfredley committed:
Update README to point at canonical guide on grails.apache.org (refs apache/grails-static-website#354)

jamesfredley committed:
Replace dead publisher CI with sample-app gradle check workflow

Delete: githubactions-build.sh
Add: .github/workflows/gradle.yml (auto-detects Java from Gradle wrapper version)

(refs apache/grails-static-website#354)
